### PR TITLE
Create blastSepolia.ts

### PR DESCRIPTION
**Pull request by armatrix**
## PR-Codex overview
This PR focuses on adding Blast Sepolia chain definition. 

### Detailed summary
- Added Blast Sepolia chain definition in `src/chains/definitions/blastSepolia.ts`
- Exported Blast Sepolia from `src/chains/index.ts`
- Updated `.changeset/long-plums-serve.md` with a patch for "viem"

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a changeset. https://github.com/wevm/viem/blob/main/.github/CONTRIBUTING.md#4-add-changeset

**Comment by jxom** [comment]
And also export the chain please: https://github.com/wevm/viem/blob/main/.github/CONTRIBUTING.md#4-export-your-chain

File: src/chains/definitions/blastSepolia.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export const blastSepolia = /*#__PURE__*/ defineChain({
+  id: 168_587_773,
+  name: 'Blast Sepolia',
+  nativeCurrency: {
+    name: 'Blast Sepolia Ether',
+    symbol: 'ETH',
+    decimals: 18,
+  },
+  rpcUrls: {
+    default: {
```
Comment (acoutts) [inline src/chains/definitions/blastSepolia.ts:12]: Need to add 'public' one as well

**Review by acoutts — CHANGES_REQUESTED** [review]

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a changeset following the [instructions](https://github.com/wevm/viem/blob/main/.github/CONTRIBUTING.md#4-add-changeset).

**Comment by kmjones1979** [comment]
Thanks for this!

**Comment by Solniechniy** [comment]
Hey guys @armatrix @tmm @kmjones1979 @acoutts, I'm trying to add blastSepolia to my existing dapp and faced with error while using blastSepolia. The issue that I can't do view methods when the user not connected with wallet. Everything works when wallet connected. Was trying on the 1.21 version and also with the newest release - the same issue.

**Comment by armatrix** [comment]
> Hey guys @armatrix @tmm @kmjones1979 @acoutts, I'm trying to add blastSepolia to my existing dapp and faced with error while using blastSepolia. The issue that I can't do view methods when the user not connected with wallet. Everything works when wallet connected. Was trying on the 1.21 version and also with the newest release - the same issue.

some methods has been updated in wagmi v2, assuming u r using wagmi.

**Pull request merged**